src/frontend/src/test-e2e/util.ts

```diff
@@ -191,6 +288,33 @@ export async function removeVirtualAuthenticator(
   return await browser.removeVirtualWebAuth(authenticatorId);
 }
 
+export async function getWebAuthnCredentials(
+  browser: WebdriverIO.Browser,
+  authId: string
+): Promise<WebAuthnCredential[]> {
+  return await browser.getWebauthnCredentials(authId);
+}
+
+export async function addWebAuthnCredential(
+  browser: WebdriverIO.Browser,
+  authId: string,
+  credential: WebAuthnCredential,
+  rpId: string
+): Promise<void> {
+  return await browser.addWebauthnCredential(
+    authId,
+    rpId,
+    credential.credentialId,
+    credential.isResidentCredential,
+    credential.privateKey,
+    credential.signCount
+  );
+}
+
+export function originToRpId(origin: string): string {
```

Comment:
what does RpId stand for?

Reply:
Relying Party Id (as per WebAuthn standard).